HTML/1. intro.md

```diff
@@ -0,0 +1,33 @@
+# Intro
+
+![HTML](https://img.17qq.com/images/mhsgsnffsky.jpeg)
+
+HTML is a markup language, meaning that it is used to create webpages, so it's not like a normal programming language like Python
```

Comment:
The use of "normal programming language" here suggests that HTML can be compared to programming languages, as if HTML were a programming language, just not a "normal" one, contradicting what was said before, that it is a markup language.

I'd suggest just removing the "normal."

Also, it may be better to refer to Python as a "scripting language," but whatever.